mbf_utility/include/mbf_utility/navigation_utility.h

```diff
@@ -58,38 +58,30 @@ namespace mbf_utility
  * @brief Transforms a point from one frame into another.
  * @param tf_listener TransformListener.
  * @param target_frame Target frame for the point.
- * @param target_time Time, in that the frames should be used.
  * @param timeout Timeout for looking up the transformation.
  * @param in Point to transform.
- * @param fixed_frame Fixed frame of the source and target frame.
  * @param out Transformed point.
  * @return true, if the transformation succeeded.
  */
 bool transformPoint(const TF &tf,
```

Comment:
way cleaner interface

Reply:
I would not say it's cleaner, but its simpler. But that's :+1: